gen/calc.py

```diff
@@ -381,6 +382,12 @@ def validate_dns_search(dns_search):
     assert len(dns_search.split()) <= 6, "Must contain no more than 6 domains"
 
 
+def calculate_dcos_net_cluster_name(dcos_net_cluster_identity, cluster_name):
+    return 'minuteman' if dcos_net_cluster_identity == 'false' \
+           else cluster_name[:255] if len(cluster_name) > 250  \
+           else '-'.join([cluster_name, ''.join(random.choice(string.ascii_lowercase) for _ in range(5))])
```

Comment:
Will it generate a new random cookie on each upgrade?

Reply:
Excellent catch! I guess that is the case. Let me see what is the best way to fix it